src/incmake/component_CDEPS.mk

```diff
@@ -0,0 +1,93 @@
+# Location of the ESMF makefile fragment for this component:
```

Comment:
It seems to me all the changes in the component_*,mk are for gnu make, is it correct? We are planning to get rid if gnu make in the official NEMS branch, is there a plan when we are going to move the cmake build?

Reply:
HAFS/UFSATM/HYCOM/CMEPS are all building with CMake.
https://github.com/hafs-community/ufs-weather-model/blob/feature/hafs_couplehycom_cmeps/CMakeLists.txt

CDEPS is being added to the build system.

Reply:
One more note. I did not test the make build. I only used CMake and built with NEMS,UFSATM,HYCOM,CMEPS.

Reply:
@junwang-noaa yes. you are right and these staff is done before switching to cmake. I think there is no harm to commit them also. Once gnu make is removed these part will be gone anyway.